src/Features/Dev/EnforceSingleSignOn.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace P4\MasterTheme\Features\Dev;
+
+use P4\MasterTheme\Feature;
+
+/**
+ * @see description().
+ */
+class EnforceSingleSignOn extends Feature
+{
+    /**
+     * @inheritDoc
+     */
+    public static function id(): string
+    {
+        return 'enforce_sso';
+    }
+
+    /**
+     * @inheritDoc
+     */
+    protected static function name(): string
+    {
+        return __('Enforce Single Sign On', 'planet4-master-theme-backend');
```

Comment:
We don’t want to expose this option in the backend for editors, right?

Reply:
Yes, I need to update it.